### PR TITLE
Added omniauth facebook to add ability to sign-in using facebook

### PR DESCRIPTION
**Pull request by jjangmes**
Added omniauth function. You can see it when you click sign-in. You will have the option to sign-in with Facebook. But when you click on it, you will be taken to facebook and get an error message. I think this is happening because I signed-up Facebook developer with the URL "fischer-chess.herokuapp.com", so I think that's why it's throwing that error message.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -233,6 +233,10 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :facebook, "385815088277667", "d1429b4b804fdb0c52c03128965440e8"
```
Comment (jcrides) [inline config/initializers/devise.rb]: should probably not have the secret in the file like this. you could set a heroku environment variable for it and then just use that here instead.
set the heroku variable with config:set APP_SECRET="whatever the secret is"
then in the initializer use   <%= ENV['APP_SECRET'] %>    where the secret is now

**Comment by jjangmes** [comment]
Wait. I think I accidentally merged into the master. Is there a way to un-do this?

**Comment by helloramon** [comment]
@jjangmes going back to it here: https://github.com/Team-Fischer/chess/pull/19 you should be able ot see your merge commit into master, then on the right there is a Revert option :)

**Comment by jjangmes** [comment]
great thanks ramon!

**Pull request merged**